packages/studio-plugin/src/types/PageState.ts

```diff
@@ -1,11 +1,8 @@
 import { Stream } from "@yext/pages";
-import { ComponentState } from "./ComponentState";
 import { PropValueKind } from "./PropValues";
+import { LayoutMetadata } from "./LayoutMetadata";
 
-export type PageState = {
-  componentTree: ComponentState[];
-  cssImports: string[];
-  filepath: string;
+export type PageState = LayoutMetadata & {
```

Comment:
I was under the impression that we wanted pages to exactly mirror layouts and that we'd always want to keep them in sync so that a layout and a non-PagesJS page would always be interchangeable. But the distinction you're making makes sense. For now, to be on the safe side I've removed the coupling between `PageState` and `LayoutMetadata` so the types are independent. It'll take a little more thought to see what the best way is to relate the types safely and coherently like in your example

Reply:
Okay, let's keep them independent for now. We can punt on ironing out the type hierarchy. I will make an item for that.